src/app/features/posts/components/post-card/post-card.component.ts

```diff
@@ -39,6 +39,7 @@ import {
 export class PostCardComponent implements OnInit {
   @Input() title: any;
   @Input() url: any;
+  @Input() id: any;
   @Input() animate: boolean;
```

Comment:
Animation concern is not related to business. Animate should not be a input, the animation should play on the img (load) event.